static_report/package.json

```diff
@@ -118,5 +148,8 @@
     "singleQuote": true,
     "trailingComma": "all"
   },
-  "packageManager": "yarn@3.2.1"
+  "packageManager": "yarn@3.2.1",
+  "volta": {
```

Comment:
What's this purpose?

Reply:
I believe this was auto added by my node version manager, most likely it auto writes. I'll remove this.

https://docs.volta.sh/guide/